public/app/features/templating/template_srv.ts

```diff
@@ -41,10 +41,27 @@ export class TemplateSrv {
     return this.builtIns.__interval.value;
   }
 
+  /**
+   * @deprecated: this instance variable should not be used and will be removed in future releases
+   *
+   * Use getVariables function instead
+   */
+  get variables(): any[] {
+    return this.getVariables();
+  }
+
+  getVariables = (): VariableModel[] => {
```

Comment:
does this function need an arrow function (set on the instance) ?

Reply:
no, good catch!